### PR TITLE
feat(cdn): support ipv6 in cdn domain

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

support ipv6 in cdn domain

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
support ipv6 in cdn domain
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run TestAccCdnDomain_configs'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run TestAccCdnDomain_configs -timeout 360m -parallel 4
=== RUN   TestAccCdnDomain_configs
=== PAUSE TestAccCdnDomain_configs
=== CONT  TestAccCdnDomain_configs
--- PASS: TestAccCdnDomain_configs (644.03s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       644.190s
```

### CODE REVIEW

File: huaweicloud/resource_huaweicloud_cdn_domain.go [hunk 1]
```diff
@@ -278,6 +278,10 @@ func resourceCdnDomainV1() *schema.Resource {
 						"force_redirect":             &forceRedirectAndCompress,
 						"compress":                   &forceRedirectAndCompress,
 						"cache_url_parameter_filter": &cacheUrlParameterFilter,
+						"ipv6_enabled": {
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_cdn_domain.go]: suggest to rename it to `ipv6_enable` keep consistent with [subnet](https://registry.terraform.io/providers/huaweicloud/huaweicloud/latest/docs/resources/vpc_subnet#ipv6_enable)

Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_cdn_domain.go]: fixed

**Comment by Jason-Zhang9309** [comment]
```
make testacc TEST='./huaweicloud' TESTARGS='-run TestAccCdnDomain_configs'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run TestAccCdnDomain_configs -timeout 360m -parallel 4
=== RUN   TestAccCdnDomain_configs
=== PAUSE TestAccCdnDomain_configs
=== CONT  TestAccCdnDomain_configs
--- PASS: TestAccCdnDomain_configs (668.24s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       668.422s
```

**Pull request merged**